climada/hazard/base.py

```diff
@@ -197,6 +201,23 @@ def __init__(self, haz_type='', pool=None):
         else:
             self.pool = None
 
+    @classmethod
+    def get_default(cls, attribute):
+        """Get the Hazard type default for a given attribute.
+
+        Parameters
+        ----------
+        attribute : str
+            attribute name
+
+        Returns
+        ------
+        Any
+        """
+        return {
+            'frequency_unit': DEF_FREQ_UNIT,
+        }.get(attribute)
```

Comment:
I am not sure to understand why this is needed, and if yes, then why is not needed for the other classes with a frequency unit?

Reply:
That's just for the `clear` method, to be able to restore the default. It could have been integrated right there, but I somehow liked it better outside. In case another Hazard is by default on a 'daily' base, it seems to be more natural to overwrite `get_default` then `clear`.
The other classes with a `frequency_unit` don't have a `clear` method. That's why.